### ISSUE TITLE
JS errors when using Mailgun API

### ISSUE THREAD

**Issue by tlbsixty7** [formvalidation/support#360]
Good day. I've been getting "TypeError: jQuery* is not a function" when integrating the Mailgun API with your plugin. I tried out the demonstration on your website, and I get the same errors. Please see the attached screen grab and here is the returned source causing the error:
jQuery111209042887911092483_1445014220936({
  "address": "trevor@gmai.co",
  "did_you_mean": "trevor@gmail.com",
  "is_valid": false,
  "parts": {
    "display_name": null,
    "domain": null,
    "local_part": null
  }
})

Regards,
Trevor
![mailgun-api-error](https://cloud.githubusercontent.com/assets/8731497/10548146/043464d0-73f6-11e5-9500-a3e430a3c8ba.PNG)

**Comment by larscallary** [comment]
Trevor - you ever get any resolution to this?  Just trying to do the same thing - it appears to be working - yet still throws this same error.  On my machine it doesn't error out on the demo and I tried to use the demo code so I figure I screwed something up.  Thoughts?

**Comment by tlbsixty7** [comment]
No, I didn't resolve this. Frankly, I forgot about the issue as it has been 10 months. Somewhat disappointed as I paid for the plugin, so expected some sort of resolution for a JQuery error when the plugin's built on it.

**Comment by nghuuphuoc** [comment]
@larscallary Can you check it again? I don't see any problem with the [example](http://formvalidation.io/examples/using-mailgun-api-validate-email-address/bootstrap.html)

**Comment by larscallary** [comment]
@nghuuphuoc  - here's what I'm seeing. Maybe this isn't an error?  https://www.dropbox.com/s/vmezdc4p5wi6yg5/uncaught_ref_formvaldationexample.mov?dl=0
